internal/api/resolver_mutation_performer.go

```diff
@@ -249,10 +224,9 @@ func (r *mutationResolver) PerformerUpdate(ctx context.Context, input PerformerU
 			return fmt.Errorf("performer with id %d not found", performerID)
 		}
 
-		if err := performer.ValidateDeathDate(existing, input.Birthdate, input.DeathDate); err != nil {
-			if err != nil {
-				return err
-			}
+		err = performer.ValidateDeathDate(existing, input.Birthdate, input.DeathDate)
+		if err != nil {
+			return err
 		}
```

Comment:
The original code wasn't right, but I think we should keep the `if err := ...; err != nil {` structure.